docs/group/README.md

```diff
@@ -1,4 +1,4 @@
-# Group
+s# Group
```

Comment:
remove s

Reply:
good catch, pretty sure happened when switching from my prompt to vscode